src/ontology/mondo-edit.obo

```diff
@@ -273127,6 +273157,7 @@ id: MONDO:0013904
 name: muscular dystrophy-dystroglycanopathy (congenital with brain and eye anomalies), type a, 8
 def: "Any muscular dystrophy-dystroglycanopathy, type A in which the cause of the disease is a mutation in the POMGNT2 gene." [MONDO:patterns/disease_series_by_gene]
 synonym: "MDDGA8" RELATED ABBREVIATION [MONDO:Lexical, OMIM:614830]
+synonym: "muscle-eye-brain-POMGNT2 related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/